### PR TITLE
Surfaces reserve status to user

### PR DESCRIPTION
**Pull request by ashfurrow**
Work for #1623.

### CODE REVIEW

File: Pods [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 14b762581503bdd3f6eff894dffcd4a5de153333
+Subproject commit bf70d288b028610988c006038c31bc97dc77125f
```
Comment (orta) [inline Pods]: what's going on here?

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
        
            2 Warnings
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td data-sticky="true">Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
    
    
  </tbody>
</table>
  

  



Time | Class | Function |
| --- | ----- | ----- |244.7ms | [AuctionViewController.swift](/artsy/eigen/tree/ashfurrow-reserve/Artsy/View_Controllers/Auction/AuctionViewController.swift#L181) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
188.3ms | [SwiftyJSON.swift](/artsy/eigen/tree/ashfurrow-reserve/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
169.1ms | [WebSocket.swift](/artsy/eigen/tree/ashfurrow-reserve/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
75.9ms | [Observable.swift](/artsy/eigen/tree/ashfurrow-reserve/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
73.2ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/ashfurrow-reserve/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
67.8ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/ashfurrow-reserve/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, sideMargin: String)
64.5ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/ashfurrow-reserve/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
62.4ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/ashfurrow-reserve/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)
59.4ms | [RefinementOptionsViewController+Private.swift](/artsy/eigen/tree/ashfurrow-reserve/Artsy/View_Controllers/Auction/RefinementOptionsViewController+Private.swift#L63) | func stackView() -> ORStackView




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Pods (hunk 1 above)
Comment (ashfurrow) [inline Pods]: Not sure, probably and accidental commit.

**Comment by ashfurrow** [comment]
Failure appears legit: LiveAuctionLotViewControllerTests.swift

**Comment by ashfurrow** [comment]
Forgot to check in a reference image >.< should be good for review.

**Comment by orta** [comment]
Yep - 👍

**Pull request merged**